### DISCUSSION TITLE
NetBSD featherpad package

### DISCUSSION THREAD

**Discussion by 0323pin** [tsujan/FeatherPad#763] [General]
Hi,

FI, I've just swiched the NetBSD package to build with Qt6 by default. Qt5 build is still available for source builds if an user wants to default to it.
Pre-compiled binaries on the next update will build against Qt6.

Commit message for your reference, https://mail-index.netbsd.org/pkgsrc-changes/2023/09/05/msg282037.html

My plan is to switch `feathernotes` and `kvantum` to Qt6 tomorrow.

Regards

**Comment by tsujan** [comment]
Thanks!

> My plan is to switch feathernotes and kvantum to Qt6 tomorrow.

Good idea. Personally I use all of them with Qt6.

Kvantum should be compiled against Qt5 and Qt6 simultaneously (as done in [Arch](https://archlinux.org/packages/extra/x86_64/kvantum/)) because Kvantum Manager and `kvantumpreview` are in Qt5  — they're ready for Qt6 and can be compiled against it manually, but I'll wait until Qt6 becomes the default version in rolling distros of Linux (and KDE and LXQt are ported to it).

**Comment by 0323pin** [comment]
Thanks!
I'll switch `feathernotes` and `arqiver` tomorrow, than. Will wait with `kvantum` until the next branch of stable packages, which I hope (if I manage to finish on time) will include the next November release of LXQt.

**Reply by tsujan** [reply]
BTW, LXQt has WIP Qt6 branches for its core libraries, so that it already supports Qt6 styling alongside its main release (I have them here for Qt6 Kvantum and Qt6 apps; see more recent comments of https://github.com/lxqt/lxqt/discussions/2139). But I guess you prefer to focus on latest releases.

**Reply by 0323pin** [reply]
Cheers! I'll go through the thread in detail later. The major issue for me right now, is that we don't have KF6 yet 😞 

If we did, I could take the branch for a spin but, of course for official builds, we stick to releases. Personally, I run a few things built straight from git-HEAD, my window manager, terminal and shell.

**Reply by tsujan** [reply]
> The major issue for me right now, is that we don't have KF6 yet

Oh, you're not alone. As mentioned in https://github.com/lxqt/lxqt/discussions/2139, the lack of a stable version of KF6 is the only obstacle to porting the whole LXQt to Qt6.

**Reply by 0323pin** [reply]
Yeah, I know I'm not alone but, given we release stable packages 4 times _per_ year, it's important things fall into place at the right time.

Personally, I don't really care because, I use the development branch of the OS and current packages. Unlike, what many think, it's possible to be on the bleeding edge on BSD, else I wouldn't be running `alacritty`-0.13.0, `elvish`-0.20.0 and so on ... but, I need to follow the branch rules 😄

**Comment by 0323pin** [comment]
I've managed to get `feathernotes` default migration done today, http://mail-index.netbsd.org/pkgsrc-changes/2023/09/05/msg282058.html

`arqiver` tomorrow 😴

**Comment by 0323pin** [comment]
Fixed `arqiver` also, https://mail-index.netbsd.org/pkgsrc-changes/2023/09/06/msg282082.html
